src/App.js

```diff
@@ -1,33 +1,37 @@
 import React, { Component } from 'react';
-import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
+import { MovieFilter } from './components/MovieFilter';
+import './App.scss';
 
 export class App extends Component {
-  state = {};
+  state = {
+    query: '',
+  }
+
+  queryChangeHandler = (query) => {
+    this.setState({
+      query,
+    });
+  }
```

Comment:
save value in lower case

Reply:
Fixed, but this means that all text in filter form will be in lower case. I haven't seen any search behave this way.